### PR TITLE
Add info how to make Fb login work

### PR DESCRIPTION
**Pull request by AdamZaczek**

### CODE REVIEW

**Comment by langpavel** [comment]
Hi @AdamZaczek. Thank you for effort! I'm not good in English but I think your text should be reworded a little. Can anyone help?

**Comment by AdamZaczek** [comment]
I can take a bit more time tomorrow to make it more documentation alike.

2017-05-06 0:40 GMT+02:00 Pavel Lang <notifications@github.com>:

**Comment by AdamZaczek** [comment]
@langpavel I improved wording in this pull request and removed a typo. Can We merge ;>?

File: docs/getting-started.md [hunk 1]
```diff
@@ -167,3 +167,15 @@ $ git fetch react-starter-kit
 $ git merge react-starter-kit/master
 $ yarn install
 ```
+
+### How to Add Facebook Login Work
+###### 1. Visit https://developers.facebook.com and create account if you don't already have one.  
+###### 2. Add a new app.  
+###### 3. In your application's folder create an .env file and copy App ID and App secret values from developers.facebook. Your .env file should look similar to this:
```
Comment (langpavel) [inline docs/getting-started.md]: You are using level 6 header instead of list item (` - `), this is not good I think..

Comment (AdamZaczek) [inline docs/getting-started.md]: Not a problem, fixed this one ;) Let me know if it's k.

**Comment by AdamZaczek** [comment]
Updated changes as requested.

**Comment by langpavel** [comment]
Ok, thanks @AdamZaczek, delegating to @frenzzy or @koistya

**Comment by AdamZaczek** [comment]
@koistya @frenzzy Hey guys, is it mergable ;>?

**Comment by AdamZaczek** [comment]
@langpavel Can you approve the pull or does it have to be @frenzzy or @koistya ?

File: docs/getting-started.md [hunk 2]
```diff
@@ -167,3 +167,15 @@ $ git fetch react-starter-kit
 $ git merge react-starter-kit/master
 $ yarn install
 ```
+
+### How to Add Facebook Login Work
```
Comment (langpavel) [inline docs/getting-started.md]: Uhmm, sorry, I don't read this carefully. This title will need reword. *Add … Work* sounds weird.

File: docs/getting-started.md [hunk 3]
```diff
@@ -167,3 +167,15 @@ $ git fetch react-starter-kit
...
+- Visit https://developers.facebook.com and create account if you don't already have one.  
+- Add a new app.  
+- In your application's folder create an .env file and copy App ID and App secret values from developers.facebook. Your .env file should look similar to this:
```
Comment (langpavel) [inline docs/getting-started.md]: `dotenv` is not part merged in master so you cannot refer `.env` file.

Comment (DaveyEdwards) [inline docs/getting-started.md]: @langpavel you are correct add/work is not quite proper.  For the title either: 
How to add Facebook Login
or 
How to get Facebook Login working

**Comment by AdamZaczek** [comment]
Ok, I will rename those today.

2017-06-13 4:44 GMT+02:00 Davey <notifications@github.com>:

**Comment by langpavel** [comment]
@AdamZaczek This is really long opened, so..
Please, reopen this ASAP, but when it was ready and you will have the time for this really interesting aspect of the app.

**Pull request closed without merging**